rollup/internal/controller/relayer/l2_relayer.go

```diff
@@ -344,8 +344,9 @@ func (r *Layer2Relayer) ProcessGasPriceOracle() {
 			expectedDelta = 1
 		}
 
-		// last is undefine or (suggestGasPriceUint64 >= minGasPrice && exceed diff)
-		if r.lastGasPrice == 0 || (suggestGasPriceUint64 >= r.minGasPrice && (suggestGasPriceUint64 >= r.lastGasPrice+expectedDelta || suggestGasPriceUint64+expectedDelta <= r.lastGasPrice)) {
+		// last is undefined or (suggestGasPriceUint64 >= minGasPrice && exceed diff)
+		if r.lastGasPrice == 0 || (suggestGasPriceUint64 >= r.minGasPrice &&
```

Comment:
lgtm. could you also update https://github.com/scroll-tech/scroll/blob/v4.4.83/rollup/internal/controller/relayer/l1_relayer.go#L254 and https://github.com/scroll-tech/scroll/blob/v4.4.83/rollup/internal/controller/relayer/l1_relayer.go#L261? if for consistency.